File: test/golden/request/ruby/2.rb
# Built by LucyBot. www.lucybot.com
uri = URI("http://undefined/")
http = Net::HTTP.new(uri.host, uri.port)
uri.query = URI.encode_www_form({
  "q" => [
    "one",
    "two",
    "three"
  ]
})
request = Net::HTTP::Get.new(uri.request_uri)
@result = http.request(request).body
puts @result

File: test/golden/request/php/2.php
// Built by LucyBot. www.lucybot.com
$curl = curl_init();
curl_setopt($curl, CURLOPT_RETURNTRANSFER, 1);
$query = array(
  "q" => array(
    0 => "one",
    1 => "two",
    2 => "three"
  )
);
curl_setopt($curl, CURLOPT_URL,
  "http://undefined/" . "?" . http_build_query($query)
);
$result = curl_exec($curl);
echo $result;
